README.md

```diff
@@ -248,6 +248,10 @@ You __must__ use the label key `method` to retrieve a counter for a particular R
 rpc_method_calls_total{method="starknet_getStateUpdate"}
 rpc_method_calls_failed_total{method="starknet_chainId"}
 ```
+You may also use the label key `version` to specify a counter for a particular version of the RPC API, for example:
```

Comment:
Otherwise it sounds like you're creating a counter for a version.
```suggestion
You may also use the label key `version` to specify a particular version of the RPC API, for example:
```